src/main/java/org/group2/textprocessingtool/controllers/TextProcessingMainController.java

```diff
@@ -2,15 +2,19 @@
 
 import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
-import javafx.scene.control.Alert;
-import javafx.scene.control.TextArea;
-import javafx.scene.control.TextField;
-import javafx.scene.control.TextInputDialog;
+import javafx.geometry.Insets;
+import javafx.scene.control.*;
+import javafx.scene.layout.GridPane;
 import javafx.stage.FileChooser;
```

Comment:
Please are you sure this ought to be removed?? are they not being used on other branches?

Reply:
> Please are you sure this ought to be removed?? are they not being used on other branches?

![Screenshot (337)](https://github.com/user-attachments/assets/aa7a60b1-b095-471e-8c1d-c26ba11d310b)